config/locales/en.yml

```diff
@@ -103,8 +108,6 @@ en:
       country: Country
       postcode: Postcode
       address_unknown: I don’t know where they currently live
-      residence_requirement_met:
```

Comment:
Same here, if we can have an old version (the removed one) and a new version (adding _options) like we have in `PERSONAL_DETAILS_FIELDS` then it is shared between all the people (applicant, respondent, other parties...) as this is used in multiple places.